deployment/terraform/assets/dashboard_data.json

```diff
@@ -342,16 +342,16 @@
       "steppedLine": false,
       "targets": [
         {
-          "expr": "increase(mattermost_http_requests_total{instance=~\"$server\"}[1m])",
+          "expr": "rate(mattermost_http_requests_total{instance=~\"$server\"}[1m])",
```

Comment:
From what I understand, this should be "increase". Or was I wrong the first time ? I thought you wanted this changed @ethervoid ? I keep forgetting the reason.

Reply:
Yes. The increase function is syntactic sugar and should be used in panels. Rates are usually mean for alerts or slow-moving counters as you can see in [their docs](https://prometheus.io/docs/prometheus/latest/querying/functions/#rate):

```
rate should only be used with counters. It is best suited for alerting, and for graphing of slow-moving counters.
```

In case you use `rate` your Y-axis is going to reflect the increase ration instead of the real measurement (requests, messages, etc). From their docs:

```
 It is syntactic sugar for rate(v) multiplied by the number of seconds under the specified time range window, and should be used primarily for human readability
```

Reply:
I am fine with using `increase` but then we need to divide by 60 to get the requests per second.

Reply:
Yep given that the number returned is based on the time range you're using. But is better than using rate IMHO

Reply:
Let's use rate. It does not make much sense to use increase and again divide by 60. And this is why the RPS numbers were wrong. We weren't really pushing 200k RPS then. Which makes sense now.

Reply:
Yeah, not sure what's the difference if indeed `increase` is just another `form` of doing `rate`. It's not like it's more precise is it?

Reply:
So .. just to clarify is rate different than increase/60 ?

Reply:
I believe it's the same if you are applying to [1m] which we are.

Reply:
Well, okay, then I don't see any reason _not_ to use rate.

Reply:
After investigating with Claudio looks like using rate is the way to go in this case (RPS) because using rate you get the per-second value of that instant point without aggregation

Sorry for the confusion